tests/brun/ash-c.txt

```diff
@@ -1,3 +1,2 @@
 brun -c '(ash (q . 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000007) (q . 0x0000000000000000000000000000000000000000000000000000000000000001))'
-cost = 76
-14
+FAIL: ash requires int32 args (with no leading zeros) 0x0000000000000000000000000000000000000000000000000000000000000001
```

Comment:
These new size restrictions for args 2&3 of substr and arg 2 of ash & lsh are meant to apply in all modes (safe & develop), right?

That seems correct to me.

Reply:
that's right